flac.md

```diff
@@ -443,6 +445,13 @@ Value | Picture type
 19    | Band or artist logotype
 20    | Publisher or studio logotype
 
+In case not a picture but an URI is contained in this block, the following points apply:
+
+- In case an URI does not include a protocol specifier, the URI is considered relative to the FLAC file being processed. Such URIs are retrieved with the same protocol as the FLAC file being processed, if the protocol has a concept of relative URIs.
```

Comment:
```suggestion
- The URI can be either in absolute or in relative form. In case an URI is in relative form, it is related to the URI of the FLAC content processed.
```

See [RFC 3986](https://www.rfc-editor.org/rfc/rfc3986), (no protocols, but schemes; URI is already defined with both concepts and name them absolute and relative, see abstract).

Also the concept of "file" may not exist (real time broadcasting etc).